src/SelfService/Application/PlatformDataApiRequesterService.cs

```diff
@@ -0,0 +1,146 @@
+using System.Text;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using SelfService.Domain.Exceptions;
+using SelfService.Domain.Models;
+using SelfService.Domain.Queries;
+using SelfService.Infrastructure.Persistence;
+using SelfService.Infrastructure.Persistence.Queries;
+
+namespace SelfService.Application;
+
+public class PlatformDataApiRequesterService : IPlatformDataApiRequesterService
+{
+    private class PlatformDataApiTimeSeries
+    {
+        [JsonPropertyName("timestamp")]
+        public DateTime TimeStamp { get; set; }
+
+        [JsonPropertyName("value")]
+        public float Value { get; set; }
+
+        [JsonPropertyName("tag")]
+        public string Tag { get; set; } = "";
+    }
+
+    private string GetTimeSeriesUrl()
```

Comment:
Is there a reason this is not just a constant?
I assume some timing issues, but having a parameter-less function returning a string that will never change during the lifetime of this class instance seems ripe for optimization.
This is readable and fine, though, so no big issue.